tests/integration/targets/aci_epg/tasks/main.yml

```diff
@@ -32,6 +32,7 @@
     use_proxy: '{{ aci_use_proxy | default(true) }}'
     output_level: debug
     monitoring_policy: check
+    custom_qos_policy: check
```

Comment:
@tvarohohlavy 
custom_qos_policy needs to be included in aci_epg_monitoring_policy as well for this to work. Please follow the same procedure as aci_epg and include this parameter in aci_epg_monitoring_policy

Reply:
We may need to add a check here as well if the custom_qos_policy is present only in the newer versions.
Please follow the comments posted in: https://github.com/CiscoDevNet/ansible-aci/pull/137